DATA/FX/beam_effects.ini

```diff
@@ -722,6 +722,28 @@ head_texture = ball
 trail_texture = thin
 flash_size = 3
 
+[BeamBolt]
+nickname = pi_particle_05_beam
+tip_length = 5
+core_length = 50
+tail_length = 20
+head_width = 3
+core_width = 3
+sec_core_width = 1
+tip_color = 255, 0, 0
+core_color = 0, 0, 255
+outter_color = 0, 0, 180
+sec_core_color = 0, 0, 190
+sec_outter_color = 0, 0, 255
+tail_color = 0, 0, 180
+head_brightness = 1
+trail_brightness = 0.3
+head_texture = ball
+trail_texture = wide
+flash_size = 3
+
+; ATRAX Xenos Heavy Particle Weapon

```

Comment:
Would be nice to have the comments actually at the right spot above the thing, and not below.